codingtovictory/day6/takeHomeChallenge/index.html

```diff
@@ -0,0 +1,25 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="FirstA">
+    <a href=https://www.youtube.com/watch?v=--khbXchTeE">Checkout this GPT-4 video.</a>
```

Comment:
try using the a:blank emmet vs the a emmet in VS Code and see the differences when you click the link to see how it opens. See which option you would want the user to have for an external link.